### DISCUSSION TITLE
Indonesia Translation

### DISCUSSION THREAD

**Discussion by adityaharist** [vuejs-translations/guidelines#37] [General]
Is there translation to bahasa? I think this project very usefull for indonesian who wanna learn vuejs.

**Comment by mcanam** [comment]
sepertinya belum ada

**Comment by risesia** [comment]
There is! https://docs.vuejs.id/v2/guide/ for 1.x and 2.x, 3.x isn't out yet. Here is the repo https://github.com/vuejs-id/docs

**Comment by babylittlepony** [comment]
haloo, saya siap bantu translate

**Comment by crispy-broccoli** [comment]
Ayo kita bikin team utk projek ini! Aku pengin banget

**Comment by danangace** [comment]
Apakah sudah ada team yang koordinir teman-teman? Harusnya translation ini penting bagi temen-temen programmer di Indonesia

**Reply by danangace** [reply]
Sepertinya bisa dilihat disini ya teman-teman jika ingin kontribusi -> https://github.com/vuejs-id/docs-next

**Reply by Abizrh** [reply]
@danangace , tapi kenapa nggak ada di official vuejs nya kak? [vuejs translation page](https://vuejs.org/translations/)

**Reply by danangace** [reply]
@Abizrh kayaknya belum di embed aja mgkn ya, tapi secara progress harusnya masih lanjut

**Comment by mrtampan** [comment]
i hope vuejs official can make repository vuejs translation indonesia

**Comment by rizqisetiaji7** [comment]
Saya tertarik buat ngetranslate, tapi saya masih pemula sebagai kontributor, mungkin kalau ada koordinator untuk melanjutkan proyek ini, saya sangat terbantu

**Reply by rizqisetiaji7** [reply]
<img width="1366" height="768" alt="Screenshot (7)" src="https://github.com/user-attachments/assets/8ff97e48-8971-4943-a2f0-a9573e5b5f24" />
<img width="1366" height="768" alt="Screenshot (8)" src="https://github.com/user-attachments/assets/bdf5eab7-3dc8-44d8-9825-6f38eda920e9" />

Saya hanya mencobanya sebisa saya di lokal komputer sendiri

**Comment by muamdvlpr** [comment]
ini masih dilanjut kah?
